docs/tutorials/getting_started/interactions.rst

```diff
@@ -0,0 +1,34 @@
+Accessing the interactions
+==========================
+
+This tutorial will show you briefly how to access the detailed interaction
+results corresponding to the tournament.
+
+As shown in :ref:`getting-started` let us create a tournament except that,
+this time, we will set the with_interactions parameter. (We will also reduce
+the number of turns per match to 3 and we'll run only a single repetition)::
+
+    >>> import axelrod as axl
+    >>> strategies = [
+    ...     axl.Cooperator(), axl.Defector(),
+    ...     axl.TitForTat(), axl.Grudger()]
+    >>> tournament = axl.Tournament(
+    ...     strategies, turns=3, repetitions=1, with_interactions=True)
+    >>> results = tournament.play()
+
+We can now view the detailed interactions that occurred between the players
+for each turn in each match. (The interactions are stored within a dictionary,
+so we'll add a sort to show the entries in a sensible order).::
+
+    >>> for key in sorted(tournament.interactions):
+    ...     print('%s: %s' % (key, tournament.interactions[key]))
+    (0, 0): [('C', 'C'), ('C', 'C'), ('C', 'C')]
```

Comment:
Any scope in using a tuple of players as the key?

On Thu, 3 Mar 2016, 17:12 Owen Campbell, <notifications@github.com> wrote:

> In docs/tutorials/getting_started/interactions.rst
> <https://github.com/Axelrod-Python/Axelrod/pull/474#discussion_r54912463>:
>
> > +
> > +    >>> import axelrod as axl
> > +    >>> strategies = [
> > +    ...     axl.Cooperator(), axl.Defector(),
> > +    ...     axl.TitForTat(), axl.Grudger()]
> > +    >>> tournament = axl.Tournament(
> > +    ...     strategies, turns=3, repetitions=1, with_interactions=True)
> > +    >>> results = tournament.play()
> > +
> > +We can now view the detailed interactions that occurred between the players
> > +for each turn in each match. (The interactions are stored within a dictionary,
> > +so we'll add a sort to show the entries in a sensible order).::
> > +
> > +    >>> for key in sorted(tournament.interactions):
> > +    ...     print('%s: %s' % (key, tournament.interactions[key]))
> > +    (0, 0): [('C', 'C'), ('C', 'C'), ('C', 'C')]
>
> Here's another idea - instead of holding the interactions within the
> tournament, I could just hold the match objects themselves. That way, user
> could do whatever they liked - including calls to the sparkline function!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Axelrod-Python/Axelrod/pull/474/files#r54912463>.
>